### PR TITLE
Fix depwarn on 0.5 and 0.6

### PR DESCRIPTION
**Pull request by yuyichao**

### CODE REVIEW

**Comment by yuyichao** [comment]
@dcjones Please enable travis on this repo. This PR passes tests on my travis account. https://travis-ci.org/yuyichao/Zlib.jl/builds/150169059

**Comment by yuyichao** [comment]
Ping @dcjones

**Comment by ranjanan** [comment]
Bump! @dcjones @tkelman

File: test/runtests.jl [hunk 1]
```diff
@@ -38,7 +38,7 @@ data = Any[
     "julia",
     rand(5),
     rand(3, 4),
-    sub(rand(10,10), 2:8,2:4)
+    Compat.view(rand(10,10), 2:8,2:4)
```
Comment (tkelman) [inline test/runtests.jl]: isn't this 0.8.4?

**Comment by tkelman** [comment]
No idea who has commit access here. I don't.

File: test/runtests.jl (hunk 1 above)
Comment (yuyichao) [inline test/runtests.jl]: Yes and done

**Comment by tkelman** [comment]
@ranjanan @yuyichao which packages still depend on this one that haven't migrated to its replacement? We can make a copy of this repo somewhere like JuliaIO and change metadata to point to it like we did with one other of @dcjones' packages if needed.

**Comment by ranjanan** [comment]
WinRPM depends on this AFAIK

**Comment by tkelman** [comment]
Yep, looks like it does. Worth looking into how hard it would be to migrate.

**Comment by ranjanan** [comment]
By replacement, do you mean this: https://github.com/BioJulia/Libz.jl ?

**Comment by tkelman** [comment]
That's what this package's readme says.

**Comment by yuyichao** [comment]
So what should we do to this package?

**Comment by tkelman** [comment]
migrate any of its dependents that matter to better maintained replacements. that's more worth our time than the backup option of forking it to JuliaIO and redirecting metadata to point to the fork, but if it's obsolete then why bother?

**Comment by samoconnor** [comment]
Bump

**Comment by dcjones** [comment]
Let me know if there's anyone I can add as collaborator. Obviously I'm not activity maintaining this right now. I would recommend switching to Libz.jl if you can, and let me know if there are important missing features in that library.

**Comment by samoconnor** [comment]
@fhs says Libz.jl is missing "support for raw deflate compression/decompression": https://github.com/fhs/ZipFile.jl/issues/29#issuecomment-256548749

**Comment by dcjones** [comment]
Ok, no problem: https://github.com/BioJulia/Libz.jl/commit/15f0644a91d20ef4469b00f529e6816fb5a6e7bd

**Pull request merged**